airbyte-oauth/src/main/java/io/airbyte/oauth/flows/LeverOAuthFlow.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.oauth.flows;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.collect.ImmutableMap;
+import io.airbyte.commons.json.Jsons;
+import io.airbyte.config.persistence.ConfigRepository;
+import io.airbyte.oauth.BaseOAuthFlow;
+import org.apache.http.client.utils.URIBuilder;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.net.http.HttpClient;
+import java.util.Map;
+import java.util.UUID;
+import java.util.function.Supplier;
+
+public class LeverOAuthFlow extends BaseOAuthFlow {
+
+  private static final String AUTHORIZE_URL = "https://sandbox-lever.auth0.com/authorize";
```

Comment:
This URL is only for sandbox no? It wouldn't work for a customers actual account

Reply:
We need to schedule a meeting with them so they do QA in our integration, where we would demo it live against the sandbox environment. Will discuss this with Karina.

Reply:
fixed